2025-03-26 23:54:56 - Exception in searchVets: Test
2025-03-27 12:22:21 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 13:00:20 - Exception in findById: User not found
2025-03-27 13:00:21 - Exception in findById: User not found
2025-03-27 13:00:22 - Exception in findById: User not found
2025-03-27 13:00:27 - Exception in findById: User not found
2025-03-27 14:07:12 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 14:12:00 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 14:25:01 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 14:25:43 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 15:51:07 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 15:52:48 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 15:53:34 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 15:54:50 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 15:55:00 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 16:25:05 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 16:46:17 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 16:48:40 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 17:26:41 - Exception in loadUserByUsername: User not found
2025-03-27 17:26:46 - Exception in loadUserByUsername: User not found
2025-03-27 17:26:55 - Exception in loadUserByUsername: User not found
2025-03-27 17:26:58 - Exception in loadUserByUsername: User not found
2025-03-27 17:28:30 - Exception in loadUserByUsername: User not found
2025-03-27 18:19:11 - Exception in changeRole: No enum constant com.petMed.user.model.Role.Select role
2025-03-27 18:40:08 - Exception in changeRole: No enum constant com.petMed.user.model.Role.
2025-03-27 18:55:46 - Exception in changeRole: No enum constant com.petMed.user.model.Role.
2025-03-27 18:55:48 - Exception in changeRole: No enum constant com.petMed.user.model.Role.
2025-03-27 20:47:18 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 20:47:27 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 20:55:49 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 20:55:56 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 20:56:02 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 21:03:29 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 21:03:34 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 21:03:39 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 21:03:50 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 21:03:56 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 21:56:03 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 22:29:54 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-27 23:06:23 - Exception in findById: User not found
2025-03-27 23:06:24 - Exception in findById: User not found
2025-03-27 23:06:25 - Exception in findById: User not found
2025-03-27 23:06:25 - Exception in findById: User not found
2025-03-27 23:06:26 - Exception in findById: User not found
2025-03-27 23:06:26 - Exception in findById: User not found
2025-03-27 23:06:27 - Exception in findById: User not found
2025-03-27 23:06:29 - Exception in findById: User not found
2025-03-27 23:29:01 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-28 00:16:58 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-28 00:17:13 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-28 00:17:22 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-28 00:17:55 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-28 02:29:26 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-28 02:30:30 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-28 13:34:04 - Exception in sendConfirmRegistrationEmail: Connection refused: no further information executing POST http://localhost:8081/api/notifications/send-email
2025-03-28 13:35:57 - Exception in sendConfirmRegistrationEmail: Connection refused: no further information executing POST http://localhost:8081/api/notifications/send-email
2025-03-28 13:59:47 - Exception in sendConfirmRegistrationEmail: Connection refused: no further information executing POST http://localhost:8081/api/notifications/send-email
2025-03-29 13:12:31 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-29 15:46:24 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-29 16:09:40 - Exception in findById: User not found
2025-03-29 16:09:41 - Exception in findById: User not found
2025-03-29 16:09:42 - Exception in findById: User not found
2025-03-29 16:09:42 - Exception in findById: User not found
2025-03-29 16:09:42 - Exception in findById: User not found
2025-03-29 16:09:42 - Exception in findById: User not found
2025-03-29 16:09:42 - Exception in findById: User not found
2025-03-29 16:10:55 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-29 16:34:33 - Exception in findById: User not found
2025-03-29 16:34:37 - Exception in findById: User not found
2025-03-29 16:34:43 - Exception in findById: User not found
2025-03-29 16:34:45 - Exception in findById: User not found
2025-03-29 16:36:30 - Exception in findById: User not found
2025-03-29 16:40:13 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-29 16:56:12 - Exception in createMedicalRecord: could not execute statement [Data truncation: Data too long for column 'diagnosis' at row 1] [insert into medical_records (created_on,diagnosis,pet_id,treatment,id) values (?,?,?,?,?)]; SQL [insert into medical_records (created_on,diagnosis,pet_id,treatment,id) values (?,?,?,?,?)]
2025-03-29 18:00:40 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-29 18:35:02 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-29 19:24:08 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-29 19:26:37 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-29 19:47:31 - Exception in findAllAppointmentsInfoByDayAndVet: Cannot invoke "com.petMed.pet.model.Pet.getDateOfBirth()" because the return value of "com.petMed.appointment.model.Appointment.getPet()" is null
2025-03-29 19:48:17 - Exception in findAllAppointmentsInfoByDayAndVet: Cannot invoke "com.petMed.pet.model.Pet.getDateOfBirth()" because the return value of "com.petMed.appointment.model.Appointment.getPet()" is null
2025-03-29 19:48:47 - Exception in loadUserByUsername: User not found
2025-03-29 19:49:07 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-29 19:49:59 - Exception in findAllAppointmentsInfoByDayAndVet: Cannot invoke "com.petMed.pet.model.Pet.getDateOfBirth()" because the return value of "com.petMed.appointment.model.Appointment.getPet()" is null
2025-03-29 19:50:26 - Exception in loadUserByUsername: User not found
2025-03-29 19:50:39 - Exception in findAllAppointmentsInfoByDayAndVet: Cannot invoke "com.petMed.pet.model.Pet.getDateOfBirth()" because the return value of "com.petMed.appointment.model.Appointment.getPet()" is null
2025-03-29 19:55:27 - Exception in findAllAppointmentsInfoByDayAndVet: Cannot invoke "com.petMed.pet.model.Pet.getDateOfBirth()" because the return value of "com.petMed.appointment.model.Appointment.getPet()" is null
2025-03-29 19:55:55 - Exception in findAllAppointmentsInfoByDayAndVet: Cannot invoke "com.petMed.pet.model.Pet.getDateOfBirth()" because the return value of "com.petMed.appointment.model.Appointment.getPet()" is null
2025-03-29 19:56:32 - Exception in findAllAppointmentsInfoByDayAndVet: Cannot invoke "com.petMed.pet.model.Pet.getDateOfBirth()" because the return value of "com.petMed.appointment.model.Appointment.getPet()" is null
2025-03-29 20:00:06 - Exception in findAllAppointmentsInfoByDayAndVet: Cannot invoke "com.petMed.pet.model.Pet.getDateOfBirth()" because the return value of "com.petMed.appointment.model.Appointment.getPet()" is null
2025-03-29 20:04:06 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-29 20:16:00 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-29 20:16:14 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-29 20:16:24 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-29 20:23:10 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-29 20:23:20 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-29 20:23:28 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-29 20:28:15 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-29 20:28:23 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-29 23:18:35 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-30 21:13:47 - Exception in loadUserByUsername: User not found
2025-03-30 21:15:12 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-30 22:40:06 - Exception in loadUserByUsername: User not found
2025-03-30 23:01:36 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-30 23:03:15 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-30 23:38:03 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-30 23:46:06 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-30 23:47:17 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-30 23:47:58 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-30 23:50:04 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-30 23:51:06 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-30 23:52:17 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-30 23:55:50 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 00:02:27 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 00:03:24 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 00:04:15 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 00:05:10 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 00:07:54 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 00:08:30 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 00:09:08 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 00:10:21 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 00:15:14 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 00:15:44 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 00:16:14 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 00:18:23 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 00:19:10 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 00:19:57 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 00:27:40 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 00:29:24 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 00:30:26 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 00:31:18 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 02:11:27 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 02:13:24 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 02:16:33 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 02:20:04 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 02:22:50 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 02:23:58 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 16:20:29 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 16:20:34 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 16:20:45 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 16:20:59 - Exception in sendAppointmentBookedEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 19:16:02 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 19:17:39 - Exception in sendConfirmRegistrationEmail: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 21:01:55 - Exception in sendConfirmRegistrationEmail: Read timed out executing POST http://localhost:8081/api/notifications/send-email
2025-03-31 21:13:41 - Exception in sendConfirmRegistrationEmail: [404] during [POST] to [http://localhost:8081/api/notifications/send-email] [EmailNotificationClient#sendEmail(EmailNotificationData)]: [{"timestamp":"2025-03-31T18:13:41.581+00:00","status":404,"error":"Not Found","path":"/api/notifications/send-email"}]
2025-04-01 12:53:10 - Exception in book: Read timed out executing POST http://localhost:8081/api/notifications/appointments
2025-04-01 14:13:37 - Exception in book: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/appointments
2025-04-01 14:15:36 - Exception in loadUserByUsername: User not found
2025-04-01 14:26:16 - Exception in sendNotification: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/appointments
2025-04-01 16:06:43 - Exception in registerOwner: Send failed
2025-04-01 16:13:02 - Exception in sendNotification: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/appointments
2025-04-01 16:56:13 - Exception in book: Query did not return a unique result: 2 results were returned
2025-04-01 17:03:34 - Exception in findById: User not found
2025-04-01 17:03:37 - Exception in findById: User not found
2025-04-01 17:03:38 - Exception in findById: User not found
2025-04-01 19:36:30 - Exception in loadUserByUsername: User not found
2025-04-01 19:36:34 - Exception in loadUserByUsername: User not found
2025-04-02 15:33:17 - Exception in sendNotification: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/appointments
2025-04-02 15:33:25 - Exception in sendNotification: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/appointments
2025-04-02 17:31:13 - Exception in findById: JDBC exception executing SQL [select u1_0.id,c1_0.id,c1_0.address,c1_0.city,c1_0.name,c1_0.site,u1_0.email,u1_0.first_name,u1_0.image_url,u1_0.is_active,u1_0.last_name,u1_0.password,u1_0.phone,u1_0.role,u1_0.username from users u1_0 left join clinics c1_0 on c1_0.id=u1_0.clinic_id where u1_0.id=?] [Unknown database 'pet_med'] [n/a]; SQL [n/a]
2025-04-03 10:34:59 - Exception in findById: User not found
2025-04-03 10:35:08 - Exception in findById: User not found
2025-04-03 10:35:32 - Exception in findById: User not found
2025-04-03 10:44:43 - Exception in findById: User not found
2025-04-03 10:49:58 - Exception in loadUserByUsername: User not found
2025-04-03 10:54:23 - Exception in findById: User not found
2025-04-03 11:14:14 - Exception in loadUserByUsername: User not found
2025-04-03 12:02:42 - Exception in loadUserByUsername: User not found
2025-04-03 12:26:43 - Exception in sendNotification: [500] during [POST] to [http://localhost:8081/api/notifications/appointments] [NotificationClient#sendAppointmentBookedNotification(AppointmentBookedData)]: [{"timestamp":"2025-04-03T09:26:43.661+00:00","status":500,"error":"Internal Server Error","path":"/api/notifications/appointments"}]
2025-04-03 12:26:49 - Exception in sendNotification: [500] during [POST] to [http://localhost:8081/api/notifications/appointments] [NotificationClient#sendAppointmentBookedNotification(AppointmentBookedData)]: [{"timestamp":"2025-04-03T09:26:49.063+00:00","status":500,"error":"Internal Server Error","path":"/api/notifications/appointments"}]
2025-04-03 12:27:52 - Exception in findById: User not found
2025-04-03 15:07:07 - Exception in loadUserByUsername: User not found
2025-04-03 16:27:34 - Exception in loadUserByUsername: User not found
2025-04-03 16:27:44 - Exception in sendNotification: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/appointments
2025-04-03 16:44:02 - Exception in sendNotification: Connection refused: getsockopt executing POST http://localhost:8081/api/notifications/appointments
